Update README.md (#605)
Murmele/Gittyup: README.md

`bin`, `lib`, etc.

**Build**
```
    ninja
```
    

> ### A Convenient Shell Script for Linux is available in the following location:

Reviewer: ```suggestion
### A Convenient Shell Script for Ubuntu is available [here](https://raw.githubusercontent.com/Murmele/Gittyup/master/pack/buildUbuntu.sh), and will install all the necessary prerequisites, and build a release version for immediate use.
```

Reviewer: What do you think about this? Please rename build.sh to buildUbuntu.sh

> `pack/build.sh`, and will install all the necessary prerequisites, and build a release version for immediate use.

Reviewer: ```suggestion
```

}
run_disown_silence flatpak run com.github.Murmele.Gittyup
```
How to Build with Ubuntu
-----------------

```
sudo apt install build-essential libgl1-mesa-dev
sudo apt install cmake
sudo apt install libgit2-dev
sudo apt install cmark
sudo apt install git
sudo apt install libssh2
sudo apt install openssl
sudo apt install qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools
sudo apt install qttools5-dev
sudo apt install ninja-build
#Change to your gittyup directory below

> cd ~/Nextcloud/Projects/gittyup

Reviewer: ```suggestion
cd <path to repository>/gittyup
```

git fetch
git submodule init
git submodule update
git pull
git checkout deps
cd dep/openssl/openssl/
./config -fPIC
make
#Change to your gittyup directory below

> cd ~/Nextcloud/Projects/gittyup

Reviewer: ```suggestion
cd <path to repository>/gittyup
```
